spec/version_spec.rb

```diff
@@ -33,6 +46,33 @@ def self.supported?
         end
       end
 
+      RSpec.describe Builderator::Control::Version do
+        before(:context) do
+          SCM.unregister(Test)
+          SCM.register(NoHistory)
+        end
+
+        after(:context) do
+          SCM.unregister(NoHistory)
+          SCM.register(Test)
+        end
+
+        context 'current' do
+          it 'provides an error message if no version is found' do
+            version_file = Util.relative_path('VERSION')
+            version_file.delete if version_file.exist?
+
+            expect(SCM.tags.last).to be_nil
+            expect { Version.current }.to raise_error RuntimeError
+          end
+
+          it 'falls back to VERSION file if no tags are found' do
+            Util.relative_path('VERSION').write('1.2.3')
```

Comment:
@fmitchell-r7 shouldn't we clean up the file here?

We could also stub `Util.relative_path('VERSION')` to return a mock file which prevents writing to disk.